xbmc/ContextMenus.cpp

```diff
@@ -0,0 +1,53 @@
+/*
+ *      Copyright (C) 2016 Team Kodi
+ *      http://kodi.tv
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "ContextMenus.h"
+#include "guilib/GUIWindowManager.h"
+#include "storage/MediaManager.h"
+
+
+namespace CONTEXTMENU
+{
+
+  bool CEjectDisk::IsVisible(const CFileItem& item) const
+  {
+    return item.IsRemovable() && (item.IsDVD() || item.IsCDDA());
```

Comment:
Imo here a 
<pre>
#ifdef HAS_DVD_DRIVE
  return item.IsRemovable() && (item.IsDVD() || item.IsCDDA());
#else
  return false;
#endif
</pre>

would be a good choice to 100% prevent this item to appear on builds compiled without dvd drive support.